src/main/psxsdk/libgpu/sys.c

```diff
@@ -306,8 +306,65 @@ extern void SetDrawMode(DR_MODE* p, int dfe, int dtd, int tpage, RECT* tw) {
     p->code[1] = get_tw(tw);
 }
 
-extern void SetDrawEnv(DR_ENV* dr_env, DRAWENV* env);
-INCLUDE_ASM("main/nonmatchings/psxsdk/libgpu/sys", SetDrawEnv);
+extern void SetDrawEnv(DR_ENV* dr_env_in, DRAWENV* env) {
```

Comment:
You shouldn't need extern here

Reply:
Removed it.  I wasn't sure if it was needed.  My outdated and somewhat limited C knowledge did not contain "extern function"